engine/src/main/java/com/sandboni/core/engine/sta/Context.java

```diff
@@ -42,7 +42,7 @@
     private final ConcurrentHashMap<LinkType, Boolean> adoptedLinkTypes;
 
     public boolean inScope(String actor) {
-        return filters.isEmpty() || (actor != null && filters.stream().anyMatch(actor::startsWith));
+        return filters.isEmpty() || (actor != null && filters.stream().anyMatch(actor::contains));
     }
```

Comment:
what is the use case for this change?

Reply:
Checking the path of the jar, not just the package name of the class. This will avoid the need to open the jar to see if the classes are in the scope.